packages/react-components/src/components/Picker/Picker.tsx

```diff
@@ -6,44 +6,52 @@ import { Trigger, TriggerSize } from './Trigger';
 import { IPickerListItem, PickerList } from './PickerList';
 import { Icon } from '../Icon';
 import { KeyCodes } from '../../utils/keyCodes';
+import { Tag } from '../Tag';
 
 import styles from './Picker.module.scss';
 
 const baseClass = 'picker';
 
 export interface IPickerProps {
+  className?: string;
   disabled?: boolean;
   label?: string;
   error?: string;
   options: IPickerListItem[];
-  selectedOption?: IPickerListItem;
+  selectedOptions?: IPickerListItem[];
   size?: TriggerSize;
   placeholder?: string;
   isRequired?: boolean;
   noSearchResultText?: string;
+  multiselect?: boolean;
```

Comment:
I'd avoid the `boolean` type in favor of `type` like `mode/type` with values of `single | multi`.
```suggestion
  multiselect?: boolean;
```